### DISCUSSION TITLE
osm-tag "neighbourhood" missing in default tag-mapping

### DISCUSSION THREAD

**Discussion by Meibes** [mapsforge/mapsforge#1220] [Ideas]
Hello,

I just noticed some places didn't have a label and after looking into it, it seems they are tagged as "[neighbourhood](https://wiki.openstreetmap.org/wiki/Tag:place%3Dneighbourhood)" and the default [tag-mapping](https://github.com/mapsforge/mapsforge/blob/c59f8a4c5798c0d6052b24d225fc1622c477d81e/mapsforge-map-writer/src/main/config/tag-mapping.xml#L104) does not include this tag.

So my question is, could this be added to the default tag-mapping or are there reasons why it wasn't included in the first place?

greetz

**Comment by devemux86** [comment]
I added the tag.

Mapsforge tag-mapping and themes are representative samples of common OpenStreetMap tags.
They cannot contain everything, you could customize them, if you need specific tags in your maps.

**Reply by Meibes** [reply]
thank you! I can understand that not all tags can be included, but this tag is also displayed in the openstreetmap map, so I thought it might be good to include it in mapsforge as well, to cover at least the tags that openstreetmap shows in their default map.

could you also add the label in the vtm default theme or should I start a discussion there? thanks so far =)

**Comment by devemux86** [comment]
OpenStreetMap.org uses bitmap tiles, we have vector maps.
They are quite different, so they cannot be exactly the same.

I updated both repository themes.
More customizations can be done in custom themes per app.
